service-workers/service-worker/resources/fetch-request-xhr-iframe.https.html

```diff
@@ -179,7 +179,7 @@
           '\r\n' +
           'file content\r\n' +
           '--' + boundary + '--\r\n';
-        assert_equals(response.body, expected_body);
+        assert_true(response.body === expected_body, "form data response content is as expected");
```

Comment:
@youennf Again, why is it necessary to remove `assert_equals()` in favor of `assert_true()`?